src/dbnode/persist/fs/cross_block_reader.go

```diff
@@ -24,33 +24,32 @@ import (
 	"bytes"
 	"container/heap"
 	"errors"
-	"fmt"
 	"io"
 	"time"
 
-	"github.com/m3db/m3/src/x/checked"
 	xerrors "github.com/m3db/m3/src/x/errors"
 	"github.com/m3db/m3/src/x/ident"
 	"github.com/m3db/m3/src/x/instrument"
-
-	"go.uber.org/zap"
 )
 
 var (
-	errReaderNotOrderedByIndex                = errors.New("crossBlockReader can only use DataFileSetReaders ordered by index")
-	errEmptyReader                            = errors.New("trying to read from empty reader")
-	_                          heap.Interface = (*minHeap)(nil)
+	errReaderNotOrderedByIndex     = errors.New("crossBlockReader can only use DataFileSetReaders ordered by index")
+	errUnorderedDataFileSetReaders = errors.New("dataFileSetReaders are not ordered by time")
+
+	_ heap.Interface = (*minHeap)(nil)
```

Comment:
nit: I usually like to put these right about the struct definition (i.e. `type minHeap struct`) since then you can add more in future, but is just preference so defer to whatever you want to do here.

Reply:
👍